pkgs/applications/misc/myanimelist-cli/default.nix

```diff
@@ -0,0 +1,52 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, curl
+, json_c
+, libbsd
+}:
+
+stdenv.mkDerivation rec {
+  pname = "myanimelist-cli";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "jmakhack";
+    repo = "myanimelist-cli";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-EmdkPpYEUIk9hr6rbnixjvznKSEnTCSMZz/17BfHGCk=";
+  };
+
+  buildInputs = [
+    curl
+    json_c
+    libbsd
+  ];
+
+  # support e.q. LLVM
+  postPatch = ''
+    substituteInPlace Makefile \
+      --replace 'CC:=gcc' 'CC:=${stdenv.cc.targetPrefix}cc'
+  '';
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out
+    cp -a bin $out
+    runHook postInstall
+  '';
+
+  # You have to run this with MYANIMELIST_CLIENT_ID set.
+  # Where to get one: https://myanimelist.net/apiconfig
+  # Select the type "other".
```

Comment:
Please move this to longDescription

Reply:
@pbsds Do you plan to implement this feedback?